website/www/site/content/en/powered-by/commercial/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Commercial projects"
+type: commercial
+icon: images/github-icon.svg
+---
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
```

Comment:
The icons for these are stretched weirdly for me:
![image](https://user-images.githubusercontent.com/675055/106542859-746d9c80-64b9-11eb-8384-20bcf231f174.png)

Reply:
Yeah I know, but there were the only ones I got, I will try to ask for some svg

Reply:
Thanks!